advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/__init__.py

```diff
@@ -1,3 +0,0 @@
-
-
-
```

Comment:
The last three files had loads of lines removed but none put back in as far as I can see... Is that correct?